index.html

```diff
@@ -3,12 +3,13 @@
 
  <head>
   <title>Tic Tac Toe</title>
-
+  <link type="text/css" rel="stylesheet" href="style.css">
```

Comment:
Nicely done with CSS reference.
Do the same for the JS file - how it's supposed to work otherwise